data/cities/city_data.json

```diff
@@ -1,331 +1,456 @@
 [
     {
```

Comment:
Was this regenerated something added here ? I can see ccras are different for the keyimpacts

Reply:
Yes I pulled new data from the CCRA api since Amanda made some updates there.
Because of this I also updated the city schema to include null values for the normalized_risk_score and I made a small update where we count the hazards (all related to the same change)